packages/oas/utils/json-schema-to-typescript-parsers/yamlParser.ts

```diff
@@ -1,11 +1,11 @@
-import { FileInfo } from 'json-schema-ref-parser';
-import JsonSchemaRefParser from 'json-schema-ref-parser';
+import { FileInfo, ParserError } from '@apidevtools/json-schema-ref-parser';
+import { SchemaObject } from 'ajv';
+import { load } from 'js-yaml';
 
-import { JSONSchema4 } from './JSONSchema4';
-import { applyDateFormat } from './applyDateFormat';
+import { applyDateFormat } from '../applyDateFormat';
 
-// NOTE: this is the yaml parser copied from v6.1.0 of json-schema-ref-parser (and modified) since it is not exported
-//       and this appears to be the only way to hook into parsing refs in order to fixup date fields
+// NOTE: this is the json parser copied from v9.0.9 of json-schema-ref-parser (and modified) since it is not exported
+// https://github.com/APIDevTools/json-schema-ref-parser/blob/v9.0.9/lib/parsers/yaml.js
```

Comment:
```suggestion
and this appears to be the only way to hook into parsing refs in order to fixup date fields
// https://github.com/APIDevTools/json-schema-ref-parser/blob/v9.0.9/lib/parsers/yaml.js
```
I think this part of the comment is important (it helped me remember the purpose at least :slightly_smiling_face:)